docs/example-scenario/apps/ecommerce-scenario.md

```diff
@@ -0,0 +1,152 @@
+---
+title: E-Commerce on Azure Platform as a Service
+description: Proven scenario of hosting an e-commerce site on Azure
+author: masonch
+ms.date: 7/13/18
+---
+# E-Commerce on Azure Platform as a Service
```

Comment:
This name should change.  Should be something like "Running an Ecommerce Front End on Azure"

Reply:
Based on earlier discussion, leaving out "Running" so that it's a noun phrase - Ecommerce Front End on Azure

Reply:
changed